Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml

```diff
@@ -124,6 +124,50 @@
     node: start
     deconstructionTarget: null
 
+- type: entity
+  name: Bulldog ROW
```

Comment:
This can and should be defined in `Resources/Prototypes/_Harmony/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml`